modular_skyrat/modules/microfusion/code/microfusion_designs.dm

```diff
@@ -1,228 +1,173 @@
-// CELLS AND EMITTERS
-/datum/design/basic_microfusion_cell
+// BASE FOR MCR DESIGNS
+/datum/design/microfusion
+	name = "Microfusion Part"
+	build_type = PROTOLATHE | AWAY_LATHE
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+	construction_time = 10 SECONDS //dunno if this is for mechfabs or what but I'll keep this anyway
+	category = list("Weapons")
+
+// EMITTERS
+/datum/design/microfusion/enhanced_phase_emitter
+	name = "Enhanced Microfusion Phase Emitter"
+	desc = "The core of a microfusion projection weapon, produces the laser."
```

Comment:
I recommend creating another parent called
> /datum/design/microfusion/emitter

Where it has the desc and the materials. You'll notice you have three designs that have the same desc and materials, which could be condensed with this new parent.

Reply:
What this means is that the new path for this enhanced phase emitter will be 
> /datum/design/microfusion/emitter/enhanced_phase_emitter

Reply:
I was thinking about splitting up each set of parts more, but it really isn't needed, the vars in the /microfusion/ parent apply to everything else I wanted included, the only thing I'd be knocking down is the desc var.